app/views/ckey2bibframes/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
```

Comment:
I changed the validation to:
```
validates :ckey, numericality: { message: ': You must enter a known ckey!' }
```
Also, see: https://github.com/sul-dlss/libsys-webforms/issues/392